### ISSUE TITLE
Import and export of particular tables and a whole tournament

### ISSUE THREAD

**Issue by nickkolok** [IPTnet/ipt_connect#200]
Some tournaments (e.g. regional stages of Three Science Tournament and IYPT) share the same list of problems; regional and sub-regional stages also often share the most of jurors *between the years*. There are some mechanisms to import and export jurors, but all of then should be arranged and documented.

Moreover, `ipt_connect` offers rather a high level of decentralization. It means that it is quite easy to set up a new `ipt_connect` server (and, moreover, we are working hard to make running a new tournament as simple and quick as possible, e.g. #197). Thus, it should be very easy to transfer a tournament instance from one server to another. As for now, it's easy to copy the source code (every instance is a separate Django application) and plug it into Django; however, the is no convenient way to transfer all the data between DB files.

Only tournament data should be transferred in such a way (`Participant`s, `Team`s, etc.); e.g., there is no need to transfer users and groups.

See also #70

**Comment by lysenko-nikolay** [comment]
Agree, this would ease creating every new edition's branch and spread IPT-connect to the national tournaments.
